examples/Servo-Stop/Servo-Stop.ino

```diff
@@ -0,0 +1,36 @@
+#include <Servo.h>
+#define trigPin  4
+#define echoPin  2
```

Comment:
```suggestion
byte trigPin = 4;
byte echoPin = 2;
```
Arduino's recommendation is to avoid the use of `#define` unless it's actually needed:
https://www.arduino.cc/reference/en/language/structure/further-syntax/define/

Reply:
PR Updated

Reply:
This is still not done:
>Please add a comment containing a short description of the purpose of the sketch and the required circuit. You can check the library's other examples to see the format.